### PR TITLE
(packaging) Prepare for puppetlabs-puppet_agent 1.2.0

### PR DESCRIPTION
**Pull request by MikaelSmith**
Assumes https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/101 will be merged.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
+## [1.2.0]
+
+### Summary
+Supports upgrades from puppet-agent packages! Applies to both PE and FOSS, for example upgrades from
+PE 2015.3.2 to 2015.3.3 and puppet-agent 1.3.0 to 1.4.0 are supported. Upgrading from older Puppet 3
+versions is also no longer explicitly prevented. Adds support for Solaris 11.
+
+### Known issues
+Carried-over from prior releases:
+- For Windows, trigger an agent run after upgrade to get Puppet to create the nessesary directory structures.
```
Comment (highb) [inline CHANGELOG.md]: `necessary`

File: README.markdown [hunk 2]
```diff
@@ -63,15 +69,33 @@ As part of preparing the agent for Puppet 4, the module performs several signifi
...
+  package_version => '1.4.0',
+}
+~~~
+
+This will ensure the version `1.4.0` of the puppet-agent package is installed. For version `1.4.0` and later, it will also remove the deprecated `pluginsync` setting from `puppet.conf`, unless explicitly managed elsewhere.
+
 ##Reference
 
 ###Public classes
 * [`puppet_agent`](#class-puppetagent)
```
Comment (highb) [inline README.markdown]: Woah, what happened here?

Comment (MikaelSmith) [inline README.markdown]: That's a good question... it's not present in the local source.

Comment (highb) [inline README.markdown]: Hm, seems to be fine now (for posterity, it was showing a git linenumber diff.)

**Comment by highb** [comment]
:+1:

**Pull request merged**